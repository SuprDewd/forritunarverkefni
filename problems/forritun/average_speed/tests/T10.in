00:05:10
00:12:24
00:14:49 153
00:20:05 51
00:25:58 135
00:30:56 114
00:34:45 27
00:40:42
00:40:49 166
00:41:42
00:43:34
00:50:05 2
00:50:54 152
00:52:29 67
00:54:42 79
00:59:55 197
01:07:18 23
01:12:34
01:18:13 123
01:19:30
01:19:56
01:22:51 28
01:27:13
01:34:36 51
01:42:33
01:49:21 42
01:54:48
02:01:05 90
02:01:21 185
02:07:33
02:14:35 196
02:15:48 186
02:18:32 94
