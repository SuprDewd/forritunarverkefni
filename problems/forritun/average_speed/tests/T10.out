00:05:10 0.00 km
00:12:24 0.00 km
00:40:42 39.54 km
00:41:42 42.03 km
00:43:34 47.20 km
01:12:34 104.87 km
01:19:30 109.66 km
01:19:56 110.55 km
01:27:13 118.57 km
01:42:33 128.77 km
01:54:48 138.37 km
02:07:33 162.28 km
